benchmark/king2dcompressible/ala/get_statistics

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
This script could use a comment about how it's supposed to be used (or where the description of how it's supposed to be used is located).